### PR TITLE
DM-802 - New docker registry

### PR DESCRIPTION
**Pull request by ms-darianbenito**
We will publish images to both registries fromdoppler (current) and dopplerdock (new).

On the first commit we are checking if we can remove some steps in the current proccess

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -39,59 +39,22 @@ pipeline {
...
-                    --build-arg version=int-${TAG_NAME}+${GIT_COMMIT} \\
-                    -f Dockerfile.swarm \\
-                    .'''
-                 sh '''docker build \
-                    -t "fromdoppler/doppler-webapp:qa-commit-${GIT_COMMIT}" \\
-                    --build-arg environment=qa \\
-                    --build-arg react_app_router=browser \\
-                    --build-arg public_url="/" \\
-                    --build-arg version=qa-${TAG_NAME}+${GIT_COMMIT} \\
+                    --build-arg version=int-v0.0.0+${GIT_COMMIT} \\
```
Comment (elsupergomez) [inline Jenkinsfile:50]: What happend with _production_ and _demo_?

Comment (ms-darianbenito) [reply Jenkinsfile]: It was a temporary change. Now, you can see all enviroments.

**Comment by andresmoschini** [comment]
👍

**Pull request merged**